Source/Engine/Render2D/Render2D.h

```diff
@@ -339,4 +340,29 @@ DECLARE_SCRIPTING_TYPE_NO_SPAWN(Render2D);
     /// <param name="rect">The target rectangle to draw (blurs its background).</param>
     /// <param name="blurStrength">The blur strength defines how blurry the background is. Larger numbers increase blur, resulting in a larger runtime cost on the GPU.</param>
     API_FUNCTION() static void DrawBlur(const Rectangle& rect, float blurStrength);
+
+    /// <summary>
+    /// Draws vertices array.
+    /// </summary>
+    /// <param name="t">The texture.</param>
+    /// <param name="vertices">The vertices array.</param>
+    /// <param name="uvs">The uvs array.</param>
+    API_FUNCTION() static void DrawVertices(GPUTexture* t, Array<Vector2> vertices, Array<Vector2> uvs);
```

Comment:
I would name it `DrawTexturedTriangles` and use `const Span<Vector2>& vertices` +` const Span<Vector2>& uvs` to make it more optimized (calling from C# won't allocate memory but just pass span as a pointer to fixed C# buffer).